n8n-io/n8n
# refactor(core): Separate execution `startedAt` from `createdAt`

## Summary

This PR introduces the column `execution_entity.createdAt` to differentiate between the time an execution is created in the DB and when processing begins, i.e. `execution_entity.startedAt`.

To test manually: 
- In regular mode, set `N8N_CONCURRENCY_PRODUCTION_LIMIT=1` in regular mode and activate a workflow with a cron and a wait. The `createdAt` and `startedAt` timestamps should differ.
- In scaling mode, start a main but not a worker and activate the above workflow to see all queued. Then start a worker and check the timestamps after execution completion.

As for unit tests, this change should be tested at `ActiveExecutions.add` and `JobProcessor.processJob` but those methods need heavy simplifying first, so I'm marking this as `tests-needed`. Let me know otherwise. 

## Related Linear tickets, Github issues, and Community forum posts

https://linear.app/n8n/issue/PAY-1815

## Review / Merge checklist
